packages/stream/index.js

```diff
@@ -130,18 +103,14 @@ export const makeStream = (acks, data) => {
 };
 harden(makeStream);
 
-/**
- * @template T
- * @template U
- * @template TReturn
- * @template UReturn
- * @returns {[Stream<T, U, TReturn>, Stream<U, T, UReturn>]}
- */
+// JSDoc TypeScript seems unable to express this particular function's
+// entanglement of queues, but the definition in index.d.ts works for the end
+// user.
```

Comment:
I believe one way might be to give an explicit type to the `data` and `acks` variables.

Reply:
I tried that to no avail.